src/cls/IPM/Storage/Module.cls

```diff
@@ -332,7 +332,7 @@ ClassMethod ExecutePhases(
 
         if pIsComplete {
             set tPhases = tLifecycle.GetCompletePhases(pPhases)
-            if (($listfind(tPhases, "Package") || $listfind(tPhases, "Publish")) && tModule.HaveToDeploy() && tModule.DeveloperMode) {
+            if (($listfind(tPhases, "Package") || $listfind(tPhases, "Publish")) && tModule.HaveToDeploy() ) {
                 set tPhases = $listbuild("PrepareDeploy") _ tPhases
```

Comment:
I think this should remain as we only ever want PrepareDeploy (where we install the module in a separate namespace for deployment purposes) if its dev mode and you don't want the dev namespace to be affected. If its not dev, it means this is running in a build/ci pipeline where we are going to throw away the namespace anyway after deploying